espresso-support/demo/src/androidTest/java/com/novoda/movies/MovieItemViewTalkBackTest.java

```diff
@@ -57,6 +61,18 @@ public void clickMovieItemView() {
         checkMenuDisplayed();
     }
 
+    @Test
+    public void movieItemViewHasCustomUsageHintOnClick() {
+        onView(withClassName(is(MovieItemView.class.getName())))
+                .check(matches(withUsageHintOnClick("see actions")));
+    }
+
+    @Test
+    public void movieItemViewHasCustomUsageHintOnLongClick() {
+        onView(withClassName(is(MovieItemView.class.getName())))
+                .check(matches(withUsageHintOnLongClick("open details")));
```

Comment:
could have used `withUsageHintOnLongClick(R.string.action_open_details)` but that's in title case and these matchers are strict on case.